src/platforms/ctxlink/ctxLink_mode_led.c

```diff
@@ -108,9 +108,9 @@ void mode_led_task(void)
 			 */
 		if (led_mode != mode_led_idle) {
 			/*
-				 * Set up the led control registers according to the requested mode
-				 */
-			mode_task_state = mode_led_state_on;
+						 * Set up the led control registers according to the requested mode
+						 */
```

Comment:
This comment looks over-indented - think some tabs need removing here

Reply:
Fixed.